tauri/src/endpoints.rs

```diff
@@ -60,26 +60,26 @@ pub(crate) fn handle<T: 'static>(webview: &mut WebView<'_, T>, arg: &str) -> cra
           whitelist_error(webview, error, "readBinaryFile");
         }
         WriteFile {
-          file,
+          path,
```

Comment:
Need to update the docs when we merge this in. @Laegel

Reply:
Yeah our TS API module is already referencing the `path` though.